news/_posts/2015-07-26-GSoC2015-Post-Mid-term-update.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: " GSoC2015: Post Mid-term update"
+authors: Amit
+date: "2015-07-26 13:12"
+---
+
+Hello everyone,
+
+During the past few weeks we have made some decent progress and here is a gist of updates :
+
+1. All the PR's from the previous blog post are now in, which implies we have support for
+    a. Solver `expmv` from both the packages `ExpmV.jl` and `Expokit.jl`
+    b. Liouville von Neumann Equation types and constructs.
+
+2. In addition to the above we have support for Lindblad Master Equation and most of the blog post
+   would be about the related constructs.
+
+3. There have also been few additions to `QuBase.jl`
+    a. From the previous blog, the [PR](https://github.com/JuliaQuantum/QuBase.jl/pull/37) on `vec` is now in.
+    b. We had a fix on multiplication, [PR](https://github.com/JuliaQuantum/QuBase.jl/pull/39), we had spotted this as we were testing out Lindblad based examples.
+    c. PR has been opened to support `Base.complex` function for QuArray (this requirement was spotted while working on ODE related solvers)
+
+To begin with, Lindblad Master Equation type `QuLindbladMasterEq` has fields `lindblad`, `hamiltonian` and `collapse_ops` which are the
+lindblad operator, hamiltonian and the collapse operators of the system. The lindblad operator is constructed from the function `lindblad_op`
+which accepts the hamiltonain and vector. The vector here is of type `Any` and we still have some design issues which will be summarized at the
+end, it would be great to hear from the community on the possibilities of solving these. The `liouvillian_op` is constructed from `lindblad_op`
+by passing an empty array to the vector parameter. The reason for caching Lindblad operator in type construct is that we do not need to construct it
+over and over again by the use of `operator` or other means, this turns out to be advantageous for time-independent hamiltonian cases. This is
+also consistent with the `QuLiouvillevonNeumannEq` type where we cache the `liouvillian` in the type construct itself. The two issues we
+currently have are the following :
+
+  a. Design of time dependent and time independent lindblad operator constructs, we need to think of ways to support time-dependent nature.
+  b. Promotion rule for collapse operators, as of now we cannot construct something like
+
+     ```julia
```

Comment:
Yup it was because of the indentation. Sorry for this. Fixed it and I guess it is fine now :)